src/sentry/pipeline/__init__.py

```diff
@@ -226,6 +226,12 @@ def current_step(self):
 
     def error(self, message):
         context = {"error": message}
+        extra = {
+            "organization_id": self.organization.id if self.organization else None,
+            "provider": self.provider.key,
+        }
+        logger = self.get_logger()
+        logger.error("pipeline error %s" % message, extra=extra)
```

Comment:
Should the message also be in `extra`? It'll make it easier to search in Kibana in case there are different error messages.